test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperInputExtensionsTest.cs

```diff
@@ -0,0 +1,324 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Mvc.TestCommon;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core
+{
+    /// <summary>
+    /// Test the <see cref="HtmlHelperInputExtensions" /> class.
+    /// </summary>
+    public class HtmlHelperInputExtensionsTest
```

Comment:
Hmm, meant "extension _method_". See existing `HtmlHelperHiddenTest` and `HtmlHelperPasswordTest` classes.